Resources/Prototypes/Roles/Jobs/Medical/senior_physician.yml

```
@@ -22,6 +22,8 @@
   - Maintenance
   - Chemistry
   special:
+  - !type:AddImplantSpecial
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)